spyderplugins/p_breakpoints/breakpoints.py

```diff
@@ -14,10 +14,10 @@
 
 # Local imports
 from spyderlib.baseconfig import get_translation
-_ = get_translation("p_breakpoints", dirname="spyderplugins")
+_ = get_translation("p_breakpoints", dirname="spyderplugins.p_breakpoints")

```

Comment:
THe problem I raised was with `get_translation` itself (the `dirname`argument requires a plugin name), but it's also true that I could simplify this. I'll have a look at it.